packages/terra-infinite-list/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
```

Comment:
nit: i think this should be in the "FIxed" section.

Reply:
Updated now. Thank you.